components/patient-tools/src/test/java/org/phenotips/tools/FormSectionTest.java

```diff
@@ -42,13 +53,29 @@
     String propertyName = "phenotype";
 
     @Before
-    public void setUp()
+    public void setUp() throws ComponentLookupException, NoSuchFieldException, IllegalArgumentException,
+        IllegalAccessException
     {
         List<String> categories = new LinkedList<String>();
         this.testFormSection = new FormSection(this.title, this.propertyName, categories);
         this.testFormField = mock(FormField.class);
         this.testFormGroup = mock(FormGroup.class);
         this.fieldNames = new String[] { "phenotype", "negative_phenotype" };
+        Field field = ReflectionUtils.getField(ComponentManagerRegistry.class, "cmProvider");
+        boolean isAccessible = field.isAccessible();
+        try {
+            field.setAccessible(true);
+            @SuppressWarnings("unchecked")
+            Provider<ComponentManager> cmp = mock(Provider.class);
+            field.set(null, cmp);
+            ComponentManager cm = mock(ComponentManager.class);
+            when(cmp.get()).thenReturn(cm);
+            TranslationManager tm = mock(TranslationManager.class);
+            when(cm.getInstance(TranslationManager.class)).thenReturn(tm);
+            when(tm.translate("Phenotips.FormSection.suggestTermPlaceholder")).thenReturn("");
```

Comment:
I would return "TRANSLATION1" instead of "" and correspondingly used TRANSLATION1 instead of "" a few lines below @ PT-2723: Users should be able to select what traits are reflected in the "affected" column when exporting the PED file of a pedigree